qhttp/AjaxEndpoint.cc

```diff
@@ -0,0 +1,35 @@
+// Class-header
+#include "AjaxEndpoint.hpp"
+
+// Local headers
+#include "Request.hpp"
+#include "Response.hpp"
+#include "Server.hpp"
+
+namespace lsst {
+namespace qserv {
+namespace qhttp {
+
+
+AjaxEndpoint::Ptr AjaxEndpoint::add(Server& server, std::string const& path)
+{
+    auto aep = std::shared_ptr<AjaxEndpoint>(new AjaxEndpoint);
```

Comment:
Probably should have a private ctor instead -- the intention is that they are only created by add().